Source/Core/Core/TimePlayed.cpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include <string>
+
+#include "Common/FileUtil.h"
+#include "Common/IniFile.h"
+#include "TimePlayed.h"
+
+TimePlayed::TimePlayed(std::string game_id)
+{
+  m_game_id = game_id;
+}
+
+void TimePlayed::AddTime(std::chrono::milliseconds time_emulated)
+{
+  Common::IniFile ini;
+  std::string ini_path = File::GetUserPath(D_CONFIG_IDX) + "TimePlayed.ini";
+  ini.Load(ini_path);
+
+  auto time_list = ini.GetOrCreateSection("Time Played");
+  int previous_time;
+
+  time_list->Get(m_game_id, &previous_time);
```

Comment:
I'm not entirely sure if `m_game_id` is guaranteed to be a ini-key-safe string? Might need a filter/replacement for unsafe characters.

All of these should probably also use `s64` or `u64` as the value type. You're currently using `int`, ie a 32 bit signed integer, which for milliseconds runs out in under 25 days of playtime.